homeassistant/components/garmin_connect/const.py

```diff
@@ -139,9 +142,9 @@
         None,
         False,
     ],
-    "averageStressLevel": ["Avg Stress Level", "", "mdi:flash-alert", None, True],
-    "maxStressLevel": ["Max Stress Level", "", "mdi:flash-alert", None, True],
-    "stressQualifier": ["Stress Qualifier", "", "mdi:flash-alert", None, False],
+    "averageStressLevel": ["Avg Stress Level", "lvl", "mdi:flash-alert", None, True],
```

Comment:
We should note all changes to unit of measurement as breaking changes in the PR description.

Reply:
Will document them in a moment, but a unit of measurement addition is not really breaking anything, but I understand the need for more information

Reply:
Please also mention the measurements where we have replaced empty string with None for unit of measurement.

Reply:
Will do, but I thought the outcome is the same for the end user?

Reply:
Since the value for the state attribute changes, a template that parses the value may be affected.

Reply:
Setting the unit of measurement to `None` will actually remove the corresponding state attribute from the state.